### ISSUE TITLE
Conserve the background from video

### ISSUE THREAD

**Issue by LRpz** [AliaksandrSiarohin/first-order-model#67]
It seems the default behavior of this method is to keep the background of the provided image. Would there be a way to conserve the background of the video instead? 
Thank you!

**Comment by AliaksandrSiarohin** [comment]
This would be complicated without segmentation mask, that separate background from foreground. If you would have such segmentation you can blend background from sourse with generated foreground.

**Comment by Jefersonwulf** [comment]
> This would be complicated without segmentation mask, that separate background from foreground. If you would have such segmentation you can blend background from sourse with generated foreground.

Is this segmentation mask available in any of the repository scripts?

**Comment by AliaksandrSiarohin** [comment]
No, but you can look to this repository https://github.com/zllrunning/face-parsing.PyTorch
